daemon/config/config.go

```diff
@@ -166,6 +166,7 @@ func (cfg *Config) MergeConfigurations(flagSet *pflag.FlagSet) error {
 	contents, err := ioutil.ReadFile(cfg.ConfigFile)
 	if err != nil {
 		if os.IsNotExist(err) {
+			logrus.Debugf("the %v doesn't exist: %v", cfg.ConfigFile, err)
```

Comment:
more details? such as the config file  %v doesn't exist?

Reply:
I think it should be enough to write a log in here.